src/components/ListSection.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { ListItem } from './ListItem';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTag } from '@fortawesome/fontawesome-free-solid';
+
+export const ListSection = ({ title, data, listToken, tagColor }) => {
+	return (
+		<ul>
+			<FontAwesomeIcon
+				icon={{
+					iconName: 'tag',
+					prefix: 'fas',
+				}}
+				style={{ color: tagColor, marginRight: 30 }}
+			/>
+			<h3>{title}</h3>
```

Comment:
The only elements allowed in `<ul></ul>` list are `<li>` elements. So please put the icon and subtitle above the `<ul>` :slightly_smiling_face:

Reply:
Thank you! I've fixed this. :)

Reply:
Fixed this, thanks!